py-polars/polars/functions/lazy.py

```diff
@@ -646,9 +646,9 @@ def last(*columns: str) -> Expr:
     return F.col(*columns).last()
 
 
-def nth(n: int, *columns: str) -> Expr:
+def nth(n: int | Sequence[int], *columns: str) -> Expr:
```

Comment:
Pity we have this `*columns` API, because passing `nth(1,3)` would feel really good. I think we should deprecate this API (in another PR) and make this a keyword argument, so we can pass `nth(1,3, from=['x', 'y', 'z'])`

Reply:
I was thinking the same... can follow-up with a separate improvement/deprecation.